app/web/cypress/e2e/web-functionality/get-summary.cy.ts

```diff
@@ -1,23 +1,26 @@
 // @ts-check
 ///<reference path="../global.d.ts"/>
 
+const AUTH0_USERNAME = Cypress.env('VITE_AUTH0_USERNAME') || import.meta.env.VITE_AUTH0_USERNAME;
+const AUTH0_PASSWORD = Cypress.env('VITE_AUTH0_PASSWORD') || import.meta.env.VITE_AUTH0_PASSWORD;
+const SI_WORKSPACE_URL = Cypress.env('VITE_SI_WORKSPACE_URL') || import.meta.env.VITE_SI_WORKSPACE_URL;
+const SI_WORKSPACE_ID = Cypress.env('VITE_SI_WORKSPACE_ID') || import.meta.env.VITE_SI_WORKSPACE_ID;
+const UUID = Cypress.env('VITE_UUID') || import.meta.env.VITE_UUID || "local";
```

Comment:
My Python skills are rusty, but it feels like all files should be able to import these in a new file via relative path. Thoughts?

Reply:
Absolutely, all this needs to be centralised in a cy.command so that the user doesn't need to read the exact file to import the necessary vars.

I've added some pseduo-code here as to how we might do this here but couldn't quite get it over the line in this PR:
https://github.com/systeminit/si/pull/3477/files#diff-46bd59ba837fc6b94e01a7f8aa51ed980790920e2ddb8d49702c84c7400f71fbR72-R86

Ideally these tests are simple & friendly to run so it's imperative we resolve things like the above to make them so.

Reply:
Okay sounds good. Will follow up offline for how we can track that.